### DISCUSSION TITLE
[求助]表情包手机可以用吗

### DISCUSSION THREAD

**Discussion by FeiXiao-zm** [share121/inter-knot#5060] [委托]
萌新菜菜大佬带带  看不懂啊呜呜

**Comment by Yizakl** [comment]
你的手机需要支持浏览器脚本，或者说你的手机浏览器支持扩展
安卓我不能提供意见，iOS推荐用Stay安装脚本
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Reply by FeiXiao-zm** [reply]
可恶…是安卓

**Comment by JT9364** [comment]
安卓的话，我用的是kiwi

**Reply by FeiXiao-zm** [reply]
好人！

**Comment by yihangyingzi** [comment]
via和X浏览器都自带油猴的

**Reply by FeiXiao-zm** [reply]
欸…这样子 谢谢大佬

**Comment by FeiXiao-zm** [comment]
好的，依旧看不懂  放弃了

**Reply by Yizakl** [reply]
az

**Reply by yihangyingzi** [reply]
有个绳网常见问题及解决方法的帖子里有表情包的插件，你去via浏览器或者X浏览器登绳网进去，安装这个插件就可以用了

**Reply by FeiXiao-zm** [reply]
> 有个绳网常见问题及解决方法的帖子里有表情包的插件，你去via浏览器或者x浏览器登绳网进去，安装这个插件就可以用了

好人  我去试试

**Comment by fzdqz** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/09/04/a9fbeee1e21cf1abcf7bf2e610ee91f5_8437422338490524317.png" alt="玛拉妮-气氛组" width="75" >这个表情包是我在手机上用via浏览器发的

**Reply by FeiXiao-zm** [reply]
好厉害！我弄了半天还是没搞懂放弃了😥

**Reply by fzdqz** [reply]
> 好厉害！我弄了半天还是没搞懂放弃了😥

很简单的<img src="https://bbs-static.miyoushe.com/static/2023/07/25/0302be52d1922a94378bd529cefbda6e_1187257128792751951.png" alt="诺登斯_盯" width="75" >手机下载via浏览器然后点进那个大佬发的脚本链接让它自动解析，接着开启就行了<img src="https://bbs-static.miyoushe.com/static/2024/09/10/ed624befad4ed330fbdb41c91fc09d10_3535401542533232063.png" alt="飞霄 夸夸" width="75" >

**Reply by FeiXiao-zm** [reply]
> > 好厉害！我弄了半天还是没搞懂放弃了😥
> 
> 很简单的<img alt="诺登斯_盯" width="75" src="https://camo.githubusercontent.com/ec98423a3654eb11dfe69043f7e48526e08d9034e7e280eec880603aa0fdb2b3/68747470733a2f2f6262732d7374617469632e6d69796f757368652e636f6d2f7374617469632f323032332f30372f32352f30333032626535326431393232613934333738626435323963656662646136655f313138373235373132383739323735313935312e706e67">手机下载via浏览器然后点进那个大佬发的脚本链接让它自动解析，接着开启就行了<img alt="飞霄 夸夸" width="75" src="https://camo.githubusercontent.com/2339a019bc2ac21a72dca694b50b5db9bbc7042973fd8b24287288d1ec9002d9/68747470733a2f2f6262732d7374617469632e6d69796f757368652e636f6d2f7374617469632f323032342f30392f31302f65643632346265666164346564333330666264623431633931666330396431305f333533353430313534323533333233323036332e706e67">

其实用的就是via浏览器……但是进去叫我安装拓展  看不懂😵‍💫

**Comment by Dave-12138** [comment]
稍微改了下教程
> 你至少需要一个**脚本加载器**(比如[TamperMonkey](https://www.tampermonkey.net/))或**自带脚本加载器的浏览器**(比如[via](https://viayoo.com/zh-cn/))。
然后从[这个链接](https://dave-12138.cn/static/Tampermonkey/miyoushe-emot-to-github.user.js)或者[这个链接](https://dave-12138.github.io/miyoushe-emoticon-in-discussion/miyoushe-emot-to-github.user.js)安装脚本。

**Reply by FeiXiao-zm** [reply]
好人！

**Comment by sigktt** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/baaaf420abb173dac46a466b5115e3ce.gif" alt="米游姬 感谢" width="75" >我完全了解了

**Comment by manyshoes** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/09/10/ed624befad4ed330fbdb41c91fc09d10_3535401542533232063.png" alt="飞霄 夸夸" width="75" >
安卓的话via可以，edge can版应该也行，但是这俩装了扩展也用不了用户接口，不知道为什么

**Reply by manyshoes** [reply]
确定了，edge can和via都可以，能用油猴脚本就能用表情包，至于用户接口不能用是因为我忘记把域名添加到规则里了<img src="https://img-static.mihoyo.com/communityweb/upload/22a0b41d9b75365f71f50348ef6c3233.png" alt="杰帕德 叹气" width="75" >
